src/components/Background/planetoids/Planet.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Station.css';
+
+const stationHeight = 27;
+
+function Planet(props) {
+    return (
+        <div style={{
+            display: 'block',
+            height: 'auto',
+            width: 'auto',
+            maxHeight: props.planetHeight + stationHeight,
+            maxWidth: props.planetHeight + 2 * stationHeight,
```

Comment:
Real minor general note, I usually find it helpful to put this sort of thing in brackets so the order of operations is super clear. I know that this is equivalent to `props.planetHeight + (2 * stationHeight)` but I find the unbracketed version tends to be easy for people to mix up, especially when one isn't looking too closely.

Reply:
I've added clarifying parentheses on this file and Background.js.